src/applications/check-in/pages/CheckIn.jsx

```diff
@@ -1,24 +1,48 @@
-import React from 'react';
-
-import Telephone from '@department-of-veterans-affairs/component-library/Telephone';
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import moment from 'moment-timezone';
 
 import { goToNextPageWithToken, getTokenFromRouter } from '../utils/navigation';
 
 import { checkInUser } from '../api';
 
+import BackToHome from '../components/BackToHome';
+import Footer from '../components/Footer';
+
 const CheckIn = props => {
-  const { router } = props;
+  const { router, appointment } = props;
+  const [isLoading, setIsLoading] = useState(false);
+
+  if (!appointment) {
+    goToNextPageWithToken(router, 'failed');
+    return <></>;
+  }
+
   const token = getTokenFromRouter(router);
   const onClick = async () => {
-    const json = await checkInUser({ some: 'data', token });
+    setIsLoading(true);
+    const json = await checkInUser({
+      token,
+    });
     const { data } = json;
-    if (data.status === 'checked-in') {
+    if (data.checkInStatus === 'completed') {
```

Comment:
if you haven't already, might be good to check in (heh, get it?) with @kjduensing and the CHIP team re: the official statuses they are using. would be good to be consistent from FE to vets-api to CHIP to VistA.

Reply:
That is a good idea! I will follow up.